docs/source/configuration/open-telemetry.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: OpenTelemetry
+---
+
+## OpenTelemetry
+
+The Apollo Router supports [OpenTelemetry](https://opentelemetry.io/), with exporters for:
+* [Jaeger](https://www.jaegertracing.io/)
+* [OpenTelemetry Protocol (OTLP)](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md) over HTTP or gRPC.
+
+The Apollo Router generates spans that include the various phases of serving a request and associated dependencies. 
+This is useful for showing how response time is affected by:
+* Sub-request response times.
+* Query shape (sub-request dependencies).
+* Apollo Router post-processing.
+
+Span data is sent to a collector such as [Jaeger](https://www.jaegertracing.io/) can assemble spans into a gaant chart for analysis.
+
+To get maximum benefit from distributed-tracing, all components in your system should be instrumented.
```

Comment:
maybe indicate that we send the TraceParent and TraceState headers to subgraphs by default so it will integrate automatically